### ISSUE TITLE
tensorflow=2.16 cant use GPU

### ISSUE THREAD

**Issue by yinjg1997** [tensorflow/tensorflow#64536]
### Issue type

Bug

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

source

### TensorFlow version

tf 2.16

### Custom code

Yes

### OS platform and distribution

win10

### Mobile device

win10

### Python version

3.11

### CUDA/cuDNN version

12.3

### Current behavior?

Kill  me!!!!
How exactly can I use the gpu power of tensorflow?
My python version is 3.11.
CUDA Version is 12.3.
nvcc --version is 12.3.
cudnn version is 8.9.6.50_cuda12

On this environment Pytorch's gpu is available, but tensorflow==2.16 can't utilize the gpu.

Why tensorflow just can't utilize GPU.
This problem has been tormenting me for 2 hours now, what should I do?

### Standalone code to reproduce the issue

```shell
import tensorflow as tf

print("GPU Available: ", tf.config.list_physical_devices('GPU'))

print("TensorFlow Version: ", tf.__version__)
```

**Comment by tilakrayal** [comment]
@yinjg1997,
GPU support on native-Windows is only available for 2.10 or earlier versions, starting in TF 2.11, CUDA build is not supported for Windows. For using TensorFlow GPU on Windows, you will need to build/install TensorFlow in WSL2 or use tensorflow-cpu with TensorFlow-DirectML-Plugin.
https://www.tensorflow.org/install/source_windows#install_gpu_support_optional
https://www.tensorflow.org/install/pip#windows-native

Thank you!

**Comment by yinjg1997** [comment]
Tensorflow is great project.
Not supporting running GPUs on native Windows is a disgustingly awful decision this adds context to using Tensorflow.
Thank you anyway.
